xmrstak/jconf.cpp

```diff
@@ -107,7 +107,8 @@ xmrstak::coin_selection coins[] = {
 	{ "monero7",             {cryptonight_monero, cryptonight_monero, 0u}, {cryptonight_monero, cryptonight_monero, 0u}, "pool.usxmrpool.com:3333" },
 	{ "stellite",            {cryptonight_monero, cryptonight, 3u},        {cryptonight_monero, cryptonight_monero, 0u}, nullptr },
 	{ "sumokoin",            {cryptonight_heavy, cryptonight_heavy, 0u},   {cryptonight_heavy, cryptonight_heavy, 0u},   nullptr },
-	{ "turtlecoin",          {cryptonight_lite, cryptonight_aeon, 255u},   {cryptonight_aeon, cryptonight_lite, 7u},     nullptr }
+	{ "turtlecoin",          {cryptonight_lite, cryptonight_aeon, 255u},   {cryptonight_aeon, cryptonight_lite, 7u},     nullptr },
+	{ "pinkstar",            {cryptonight_lite, cryptonight_aeon, 4u},     {cryptonight_aeon, cryptonight_lite, 4u},     nullptr }
```

Comment:
The list is in alphabetic order and it must be `{ "pinkstar",            {cryptonight_aeon, cryptonight_lite, 4u},     {cryptonight_aeon, cryptonight_lite, 7u},     nullptr }`

The order of the first bracket was wrong (the new coin algorithm must be the first value`